Sources/Commands/Error.swift

```diff
@@ -83,3 +93,11 @@ private func print(error: Any) {
         print("\(cmd): error:", error, to: &stderr)
     }
 }
+
+private func print(fix: String) {
+    if ColorWrap.isAllowed(for: .stdErr) {
```

Comment:
Eventually I want to separate out the coloring from the intent, so that code like this just applies a semantic tag and then the coloring is localized (and possibly customizable).  But that's a future improvement — for the moment, this is appropriate given the rest of the code.  One thing at a time... :)